packages/main/src/themes/sap_fiori_3_hcb/PopupBlockLayer-parameters.css

```diff
@@ -0,0 +1,3 @@
+:root {
+	--_ui5_popup_block_layer_opacity: 0.3;
```

Comment:
Add

`@import "../base/PopupBlockLayer-parameters.css";`

at the beginning of each new file.

Modify  `/sap_fiori_3_hcb/parameters-bundle.css` file to import the new file. 

`@import "./PopupBlockLayer-parameters.css";`

The same for all horizon|fiori_3|HCWB parameters-bundle.css files

Remove `-_ui5_dialog_block_layer_opacity` parameter and use the new `--_ui5_popup_block_layer_opacity` everywhere